utils/cq/daos_pylint.py

```diff
@@ -641,46 +567,16 @@ def main():
         rc_tmp.flush()
         args.rcfile = rc_tmp.name
 
-    if args.diff:
-        # There are to be two different ways of driving this, for a commit-hook where we only
-        # check files changed and in GitHub actions where we check the entire tree and are more
-        # strict for files which have changed.
-        if args.git:
-            all_files = get_git_files()
-        else:
-            all_files = FileTypeList()
-        regions = None
-        file = None
-        lineno = 0
+    if args.files_from_stdin:
+        assert not args.git, 'No longer supported'
```

Comment:
Should this be more explicit?  Something like `The --git and --files_from_stdin argument combination is no longer supported`.